src/methods.jl

```diff
@@ -305,10 +305,9 @@ end
 
 
 """
-    combine(frames::Vararg{<:AbstractArray}; method = median)
-    combine(frames::AbstractVector{<:AbstractArray}; method = median)
+    combine(frames::Vararg{<:AbstractArray{<:Number}}; method = median)
```

Comment:
The docstring here is hard to read! Putting the exact type signatures is distracting, when we can convey the same information easier. Why not put 
`combine(frames...; method=median)` and add the missing signature `combine(frames; method=median)`